pacsman/utils.py

```diff
@@ -0,0 +1,40 @@
+import dicom_numpy
+import numpy
+import png
+import scipy.ndimage
+
+
+def process_and_write_png(thumbnail_ds, png_path):
```

Comment:
I think this function could use a few tests.  In particular, the padding and scaling functionality could use some tests.  To make testing these bits easier (i.e., to avoid having to read/write png files in the tests), it may be nice to add a couple internal functions with smaller/simpler/more testable APIs.  E.g., you could have a function for determining which accepts `png_scaled` and returns `padded`.